hydra/fileformat.go

```diff
@@ -85,12 +90,18 @@ type ConvertMap struct {
 
 type RecordModifier struct {
 	convertMap    ConvertMap
+	hostnameKey   string
```

Comment:
`hostname` and `hostnameErr` are global variable currently.
If RecordModifier has `hostname` member variable, these global variables are not necessary.